spring-batch-test/src/test/java/org/springframework/batch/test/SpringBatchTestJUnit4Tests.java

```diff
@@ -46,14 +46,14 @@
 import org.springframework.test.context.junit4.SpringRunner;
 
 /**
- * Test cases for usage of {@link SpringBatchTest} annotation.
+ * Test cases for usage of {@link SpringBatchTest} annotation with JUnit 4.
  *
  * @author Mahmoud Ben Hassine
  */
 @RunWith(SpringRunner.class)
 @SpringBatchTest
-@ContextConfiguration(classes = SpringBatchTestTests.JobConfiguration.class)
-public class SpringBatchTestTests {
+@ContextConfiguration(classes = SpringBatchTestJUnit4Tests.JobConfiguration.class)
```

Comment:
Just as a side note, there is no reason to explicitly configure `JobConfiguration` here since `spring-test` will automatically find any static nested `@Configuration` class by default.

```suggestion
@ContextConfiguration
```